back-end/src/main/kotlin/Gneiss/PacketCompiler/Service/Packet.kt

```diff
@@ -1,13 +1,37 @@
 package Gneiss.PacketCompiler.Service
 
 class PacketPostRequest(val invoicePDFPath: String, val approvalPDFPath: String, val csvPDFPath: String, val compiledPDFPath: String)
+class PacketPatchRequest(val invoicePDFPath: String?, val approvalPDFPath: String?, val csvPDFPath: String?, val compiledPDFPath: String?)
 
 class PacketPostResponse()
+class PacketPatchResponse()
 
 class Packet {
     companion object {
+
+        var setInvoicePDFPath: String? = null
+        var setApprovalPDFPath: String? = null
+        var setCsvPDFPath: String? = null
+        var setCompiledPDFPath: String? = null
+
         fun packetPost(req: PacketPostRequest): PacketPostResponse {
             return PacketPostResponse()
         }
+
+        fun packetPatch(req: PacketPatchRequest): PacketPatchResponse {
+            if (req.invoicePDFPath != null) {
```

Comment:
Once we actually implement some Redis stuff here, these if statements would be perfect. If you delete those variables above, Ktlint might complain, but you could comment these lines out so we could uncomment for the future.